src/components/ui/File.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
A question: why `components/ui`?

Reply:
Why `ui` in `components` or why `ui` itself ?